.github/actions/warm-up-repo/action.yml

```diff
@@ -27,11 +22,16 @@ runs:
         cargo install cargo-quickinstall
         cargo quickinstall cargo-make --version 0.35.15
 
-    - run: yarn install
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16 ## aligned with Node version on Vercel
+        cache: yarn
+
+    - run: yarn install --immutable
       shell: bash
       env:
         PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: ${{ inputs.playwright-deps == '' }}
 
-    - run: yarn playwright install-deps ${{ inputs.playwright-deps }}
+    - run: yarn workspace @apps/site playwright install-deps ${{ inputs.playwright-deps }}
```

Comment:
Yarn Berry is stricter on package resolution, which means that we can only access a ‘binary’ (`playwright`) from a workspace that has a corresponding dependency (or dev dependency).